sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreeNode.scala

```diff
@@ -175,30 +176,7 @@ abstract class TreeNode[BaseType <: TreeNode[BaseType]]
 
   lazy val height: Int = children.map(_.height).reduceOption(_ max _).getOrElse(0) + 1
 
-  // Copied from Scala 2.13.1
-  // github.com/scala/scala/blob/v2.13.1/src/library/scala/util/hashing/MurmurHash3.scala#L56-L73
-  // to prevent the issue https://github.com/scala/bug/issues/10495
-  // TODO(SPARK-30848): Remove this once we drop Scala 2.12.
```

Comment:
You can use https://issues.apache.org/jira/browse/SPARK-30848 JIRA :-).

Reply:
Yeah, I was looking for this PR to learn about history. Thank you for the good record at that time! haha